apps/web/src/hooks/useFeatureFlags.ts

```diff
@@ -19,6 +19,16 @@ export const useIsTemplateStoreEnabled = (): boolean => {
   return isTemplateStoreEnabled ?? defaultValue;
 };
 
+export const useIsMultiProviderConfigurationEnabled = (): boolean => {
+  const value = IS_MULTI_PROVIDER_CONFIGURATION;
+  const fallbackValue = false;
+  const defaultValue = prepareBooleanStringFeatureFlag(value, fallbackValue);
+
+  const isTemplateStoreEnabled = useGetFlagByKey<boolean>(FeatureFlagsKeysEnum.IS_MULTI_PROVIDER_CONFIGURATION);
+
+  return isTemplateStoreEnabled ?? defaultValue;
+};
```

Comment:
```suggestion
export const useIsMultiProviderConfigurationEnabled = (): boolean => {
  const value = IS_MULTI_PROVIDER_CONFIGURATION_ENABLED;
  const fallbackValue = false;
  const defaultValue = prepareBooleanStringFeatureFlag(value, fallbackValue);

  const isMultiProviderConfigurationEnabled = useGetFlagByKey<boolean>(FeatureFlagsKeysEnum.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED);

  return isMultiProviderConfigurationEnabled ?? defaultValue;
};
```
Naming update and adding changes from my other suggestion.